packages/react/src/Pagination/Pagination.dev.stories.tsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react'
+import type {Meta} from '@storybook/react'
+import type {ComponentProps} from '../utils/types'
+import Pagination from './Pagination'
+
+export default {
+  title: 'Components/Pagination/Dev',
+  component: Pagination,
+} as Meta<ComponentProps<typeof Pagination>>
+
+export const DevDefault = () => (
+  <Pagination pageCount={15} currentPage={2} onPageChange={e => e.preventDefault()} showPages={{narrow: false}} />
```

Comment:
The current Pagination component doesn't seem to support any custom styling around `sx` for `style`. Is this something we want to add as part of the CSS module migration work?

Reply:
Na, if it's not supported we want to avoid adding it